### ISSUE TITLE
Model weights for MLM fine-tuning

### ISSUE THREAD

**Issue by ardakdemir** [snunlp/KR-BERT#4]
Thanks for making the models for sequence classification accessible to public!!

We wonder whether you still have the model weights for MLM training. 
Specifically, the below part of the network when using transformers library:
```
(cls): BertOnlyMLMHead(
    (predictions): BertLMPredictionHead(
      (transform): BertPredictionHeadTransform(
        (dense): Linear(in_features=768, out_features=768, bias=True)
        (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)
      )
      (decoder): Linear(in_features=768, out_features=16424, bias=True)
    )
  )
```

**Comment by snunlp** [comment]
Hello! We trained our models in a TensorFlow environment, so we don't have model weights for specific heads. We just converted the Tensorflow checkpoints of our models to those of PyTorch.

**Comment by ardakdemir** [comment]
Thanks for your reply. 
I see your point. If we want to evaluate these converted checkpoints' MLM accuracy on a new dataset with PyTorch, how should we use these checkpoints?

**Comment by snunlp** [comment]
You can use the Huggingface models like BertForPreTraining or BertForMaskeLM, by loading our model checkpoints (which are released as the PyTorch version) with the from_pretrained() method.

**Comment by ardakdemir** [comment]
Sorry for my late return. Your answer helped me solve my problem. 
Thanks a lot!

**Comment by ardakdemir** [comment]
Loading the PyTorch checkpoint with BertForMaskedLM  solves the issue.
